ui/.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+    "hooks": {
+        "pre-commit": "tsc && lint-staged"
```

Comment:
We can't use husky. As soon as you do this it will override the other pre commit hook in this repo. (So no hooks anymore for the backend) That's why I moved everything in there. Imo we can still merge this and clean up later.

Reply:
removed